automol/form/_form.py

```diff
@@ -248,18 +227,19 @@ def _sort_key(char):
     return tuple(sorted(seq, key=_sort_key))
 
 
-def argsort_symbols(seq, symbs_first=("C", "H"), symbs_last=(), idx=None):
+def argsort_symbols(
+    seq: Sequence,
+    symbs_first: Sequence[str] = ("C", "H"),
+    symbs_last: Sequence[str] = (),
+    idx: int | None = None,
+) -> tuple[str, ...]:
```

Comment:
Correction: `(seq: Sequence[str] | dict[str, int], ....) -> tuple[int, ...]`.

(Note that the return is a tuple of `int`s.)